HW_8/model.py

```diff
@@ -0,0 +1,58 @@
+
+
+def read_phonebook():
+    temp = []
+    file = open('file.txt', 'r', encoding='utf-8')
+    for line in file.read().split('\n'):
+        temp.append(line)
```

Comment:
Методы `read` и сплит не `split`, если вы используете цикл `for`. `file` – это итератор, содержащий строки в файле. `for` умеет работать с итераторами
```suggestion
    for line in file:
        temp.append(line)
```